composer.dev.json

```diff
@@ -8,16 +8,27 @@
         "phpmd/phpmd": "^2.15",
         "phpspec/prophecy-phpunit": "^2",
         "phpstan/extension-installer": "^1.3",
+        "mglaman/composer-drupal-lenient": "^1",
         "vincentlanglet/twig-cs-fixer": "^2.8"
     },
     "config": {
         "allow-plugins": {
             "dealerdirect/phpcodesniffer-composer-installer": true,
-            "phpstan/extension-installer": true
+            "phpstan/extension-installer": true,
+            "mglaman/composer-drupal-lenient": true
```

Comment:
I am using this technique https://www.drupal.org/docs/develop/using-composer/using-drupals-lenient-composer-endpoint to solve `config_filter` module compatibility issue.